library-integration-slack/src/main/kotlin/library/integration/slack/services/slack/SlackMessageClientConfiguration.kt

```diff
@@ -15,7 +16,9 @@ class SlackMessageClientConfiguration {
 
     @Bean
     fun slackMessageClient(slackSettings: SlackSettings): SlackMessageClient {
-        val slackMsgPostUrl = slackSettings.baseUrl + slackSettings.channelWebhook
+
+        val target: DynamicUrlTarget<SlackMessageClient> = DynamicUrlTarget("slack", SlackMessageClient::class
```

Comment:
You might be able to write `val target = DynamicUrlTarget(...)` because the `target`'s type (`DynamicUrlTarget<SlackMessageClient>`) should be clear to the compiler.

Reply:
fixed